lib/css/modules/_nightMode.scss

```diff
@@ -1471,9 +1475,12 @@ $title-link-visited-color: hsl(0, 0%, 65%);
 			filter: invert(100%);
 		}
 
-		.tfa-login-bottom-panel {
+		.tfa-login-bottom-panel,
+		.enable-tfa-bottom-panel {
 
-			.tfa-login-having-trouble a {
+			.tfa-login-having-trouble a,
+			.enable-tfa-learn-more a,
+			.switch-barcode-and-code a {
```

Comment:
Similar to https://github.com/honestbleeps/Reddit-Enhancement-Suite/pull/4407/files#r137137617

```
.tfa-login-bottom-panel {
    .tfa-login-having-trouble a {
         color: $link-color;
    }
}

.enable-tfa-bottom-panel {
    .tfa-login-having-trouble,
    .enable-tfa-learn-more,
    .switch-barcode-and-code {
       a { color: $link-color; }
    }
}
```